cdap-docs/included-applications/source/etl/plugins/batchsinks/s3avro.rst

```diff
@@ -0,0 +1,64 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _included-apps-etl-plugins-batch-sinks-s3avro:
+
+===================
+Batch Sinks: S3Avro
+===================
+
+.. rubric:: Description
+
+Batch sink to write to S3 in Avro format.
+
+.. rubric:: Use Case
+
+This source is used whenever you need to write to Amazon S3 in Avro format. For example,
+you might want to create daily snapshots of a database by reading the entire contents of a
+table, writing to this sink, and then other programs can analyze the contents of the
+specified file.
+
+.. highlight:: xml
+
+.. rubric:: Properties
+
+**accessID:** Access ID of the Amazon S3 instance to connect to.
+
+**accessKey:** Access Key of the Amazon S3 instance to connect to.
+
+**basePath:** The path where the data will be recorded. Defaults to the name of the dataset
```

Comment:
We need to fix the text here as well.
`The S3 path where the data is stored. Example: 's3n://logs'`